README.md

```diff
@@ -1,87 +1,134 @@
 # Tweeter
 
-Tweeter is a sample app that demonstrates how easy it is to run a Twitter-like app on DCOS.
+Tweeter is a sample service that demonstrates how easy it is to run a Twitter-like service on DCOS.
 
 Capabilities:
 
-* stores tweets in Cassandra
-* streams tweets to Kafka as they come in
-* real time tweet analytics on via Spark and Zeppelin
+* Stores tweets in Cassandra
+* Streams tweets to Kafka as they come in
+* Real time tweet analytics with Spark and Zeppelin
+
 
 ## Install and Configure Prerequisites on the Cluster
 
-You'll need a DCOS cluster with one public node and at least five private nodes.
+You'll need a DCOS cluster with one public node and at least five private nodes, DCOS CLI, and DCOS package CLIs.
+
+Install package CLIs:
+
+```base
+$ docs package install cassandra --cli
+$ docs package install kafka --cli
+```
+
+Look up the public agent IP in AWS. You need the IP of the EC2 host, not the ELB. Use this to replace `<agent_public_ip>` further down.
+
+Look up the public master IP in AWS. You need the IP of the EC2 host, not the ELB. Use this to replace `<master_public_ip>` further down.
+
+
+## Demo steps
 
-Add the Multiverse as a package source:
+Install packages for DCOS UI:
+* kafka
+* cassandra
+* zeppelin
+* marathon-lb
 
-    dcos package repo add multiverse https://github.com/mesosphere/multiverse/archive/version-2.x.zip
+Wait until the Kafka and Cassandra services are healthly.
 
-Install packages:
+## Lookup Kafka broker addresses
 
-    dcos package install --yes marathon-lb
-    dcos package install --yes cassandra
-    dcos package install --yes kafka
+Lookup the connection setting for Kafka:
```

Comment:
@brndnmtthws and I had a hallway discussion about introducing a configurable client port for the brokers. The thinking is that all brokers would use the same well known port so that clients never need to be reconfigured even in cases where the Kafka broker is reschedule to a new DC/OS agent node.

As the next step I will reach out to early adopter customers of ours to get their input.